gqueries/general/curves/ merit_participants/merit_participants_per_sector/merit_total_consumers_electricity_input_curve.gql

```diff
@@ -0,0 +1,32 @@
+- query =
+    SUM_CURVES(
+      V(
+        Q(merit_agriculture_consumers),
```

Comment:
You refer to the query merit_agriculture_consumers here, but I don't see that query anywhere. The same goes for the others.